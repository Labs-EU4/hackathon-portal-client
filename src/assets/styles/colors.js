//COLORS
export const white = '#F9F9F9';
export const black = '#010038';

//RED COLOR
export const red = '#E34449';
export const brightRed = '#F34D21';

//YELLOW COLOR
export const dimYellow = '#ffdb00';

//GREY COLOR
export const grey = '#39444D';
export const textLightGrey = '#999';
export const textDarkGrey = '#666';
export const bgLightGrey = '#f0f0f0';
export const bgGrey = '#dfe3e6';
export const borderGrey = '#39444D';
// export const lightGrey = '#F5F5F5';

//GREEN COLOR
export const lightGreen = '#afeeee';
export const green = '#5aac44';
export const brightgreen = '#25d366';
export const darkGreen = '#1b9849';


// export const primary = '#04c8de'; //--> same as the above, good combo
// export const primary = '#F5F5F5'; //--> same as the above, good combo
// export const primary = '#FF3636'; //--> same as the above, good combo
// export const primary = '#47e4bb'; //--> same as the above, good combo

//PRIMARY COLOR
export const primaryLight = '#81DBFD';
export const primary = '#03AEEF';
export const primaryDark = '#03AEEF';
// export const primary = '#8CCAE9';
// export const primaryLight = '#04c8de';
// export const primary = '#40e0d0';

//SECONDARY COLOR
export const secondaryLight = '#F8C485';
export const secondary = '#f39422';
export const secondaryDark = '#DA7C0A';
// export const secondaryDark = '#F34D21';






export const backgroundBlack = '#222';
export const backgroundWhite = '#f4f4f4';
export const lightBlue = '#0ff';

// export const white = '#f4f7f6'; // --> GOOD WHITE
export const textWhite = '#F8F9FA';
export const primaryWhite = '#eff';
export const secondaryWhite = '#eee';
export const tertiaryWhite = '#ddd';
export const quaternaryWhite = '#ccc';
export const senaryWhite = '#aaa';




// export const black = '#000';
// export const black = '#1F1C05'; --> VERY GOOD BLACK SHADE
// export const black = '#020404 '; --> ALRIGHT BLACK!!!
// export const black = '#252525'; //--> SUPER GOOD BLACK
// export const black = '#202121';
export const softGreen = '#aec';
export const spaceBlue= '#03AEEF';
export const brightPurple= '#4444dd';
export const darkTwitterTheme = '#15202B';
export const lightDarkTwitterInputBg = '253341#';
export const fbCopyCatLightTheme = '#F3F4F6';
export const youTubeColorBlue = '#065FD4';
export const flickrBlue = '#1396E7';
export const btnBlue = '#0973FF';

//Add theme using styled components

//MAIN THEME COLORS
// export const primaryColor = '#BDE6EC';
// export const primaryColor = '#aec';
// export const primaryColor = '#ffdb00';
// export const primaryColor = '#6747C7'; //--> SUPER GOOD!!
// export const primaryColor = '#f8f4ff'; //--> GOOD WHITE VERSION!!
// export const primaryColor = '#fada5e'; //--> ROYAL YELLOW
// export const lightPrimaryColor = '#FEF33E'; //--> BRIGHT COOL YELLOW
// export const primaryColor = '#FFFF01'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#F4B75D'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#448ef6'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#6decb9'; //--> VERY BRIGHT YELLOW


// export const primaryColor = '#537ec5'; //--> super good color, mostly used theme!!!!
// export const turquoise = '#ff5edf'; //--> change to secondary color

export const tertiaryColor = '#5B5F97';
export const tertiaryColorDarker = '#484b77';

export const borderGrayTheme = '#39444D';

// export const primaryColor = '#1396E7'; //--> VERY GOOOD
// export const primaryColor = '#065FD4'; //--> VERY good blue, 2nd/3rd
// export const primaryColor = '#FFf'; //--> MOST USED YELLOW!!!! 
// export const primaryColor = '#B390EE';
// export const primaryColor = '#CC2D8A';
// export const primaryColor = '#46130B';
// export const primaryColor = '#FAD52D';

// export const turquoise = '#40e0d0 ';
// export const turquoise = '#4dd599'; --> VERY GOOD
// export const turquoise = '#47e4bb'; --> EVEN BETTER
// export const turquoise = '#3fc5f0';
//DARK-MODE TOGGLE COLORS
export const sunrise = '#ffc922';
export const yellowSun = '#ffdb00';
export const blueMoon = '#003366';
export const papayawhip = 'papayawhip';


// COOL COLORS 
export const newSecondaryColor = '#B390EE';
export const newTertiaryColor = '#B0EEEE';


// DARK MODE
export const MainBackgroundColor = '#29272F';
