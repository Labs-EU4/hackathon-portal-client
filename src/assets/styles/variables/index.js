export { Solid, Gradient } from './colors';
export {
    type,
    baseFontSize,
    smallFontSize,
    h1FontSize,
    h2FontSize,
    h3FontSize,
    h4FontSize,
    button_text
} from './fonts';
export { Spacing } from './spacing';
export { media } from './media';