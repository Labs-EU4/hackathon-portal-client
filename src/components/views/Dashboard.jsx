import React from "react";
import { useDispatch } from "react-redux";

import { UserEventsDashboard } from "../templates";
import { fetchAllEvents } from "../../store/events/actions";

const Dashboard = () => {
  const dispatch = useDispatch();

  React.useEffect(() => {
    dispatch(fetchAllEvents());
  }, [dispatch]);

  return <UserEventsDashboard />;
};

export default Dashboard;
