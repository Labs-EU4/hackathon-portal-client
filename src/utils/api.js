import Axios from "axios";
import { toast } from "react-toastify";
import { resetUser } from "../store/user/actions";

const baseURL = process.env.REACT_APP_API_URL;

// if process.env.CONTEXT = "production" {
//   Point to heroku API
// }
// if process.env.BRANCH = "develop" {
  // point to Heroku staging API
// }
// if process.env.CONTEXT = "deploy-preview" {
//   point to deploy-preview HEROKU API (the separate one)
// }



export const axiosWithAuth = token =>
  Axios.create({
    baseURL,
    headers: {
      Authorization: token
    }
  });

export const axios = Axios.create({ baseURL });

export const selectToken = state => state.currentUser.token;

export const showError = message => {
  toast.error(message);
};

export const showSuccess = message => {
  toast.success(message);
};

export function* handleError({ response }, put, history) {
  const { message, statusCode } = response.data;
  if (statusCode === 404) {
    history.push("/not-found");
  }
  if (message === "jwt expired") {
    yield put(resetUser());
  } else showError(`⚠️ ${message}`);
}
