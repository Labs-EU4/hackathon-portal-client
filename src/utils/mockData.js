export const initialState = {
  events: {
    data: [
      {
        id: 1,
        event_title: "EuroHack",
        event_description: "the best event hack hack hack macbook",
        creator_id: 1,
        start_date: "10/09/11",
        end_date: "08/07/13",
        location: "Europe",
        guidelines: "cool hekhkdsjfhsdkjf skdjfhskdjfhkjsdfh skjdfhksjdfh",
        participation_type: "Team",
        category_id: 1,
        requirements: [
          {
            video_url:
              "https://res.cloudinary.com/ibkrice-com/video/upload/v1581259780/Harvesting_ifato8.mp4"
          }
        ],
        ProfileImage: [
          {
            image_url:
              "https://res.cloudinary.com/ibkrice-com/video/upload/v1581259780/Harvesting_ifato8.mp4"
          }
        ],
        rubrics: [
          "Presentation",
          "Product Fit",
          "Functionality",
          "Innovation",
          "Product Design",
          "Extensibility"
        ]
      },
      {
        id: 2,
        event_title: "World",
        event_description: "number one hacker event in the world",
        creator_id: 1,
        start_date: "07/09/11",
        end_date: "08/07/13",
        location: "Asia",
        guidelines: "come well prepared,no cheating,no fighting",
        participation_type: "Team",
        category_id: 2
      }
    ],
    categories: [
      { id: 2, category_name: "Summer Hackton" },
      { id: 3, category_name: "World Hackton" },
      { id: 3, category_name: "Asia Hackton" },
      { id: 3, category_name: "Gamer's Hackton" },
      { id: 3, category_name: "Advanced Hackton" }
    ]
  },

  currentUser: {
    token: 347534574536,
    userId: 1,
    fullname: "Jake Tom",
    email: "8omemail@google.com",
    username: "Jake22",
    bio: "I like coding ",
    image: ""
  },
  submissions: [
    {
      id: 1,
      project_title: "Mock Project",
      participant_or_team_name: "Mock Team",
      event_id: 7,
      video_url: "",
      git_url: "https://www.google.com/",
      project_writeups: "No mock write up",
      submitted_by: 14,
      created_at: "2020-03-02",
      updated_at: "2020-03-02",
      average_rating: 0,
      acted_judges: 0,
      number_of_judges: 0
    }
  ],

  teams: [
    { id: 1, team_name: "The A team", team_lead: 2 },
    { id: 2, team_name: "Hacking central", team_lead: 3 },
    { id: 3, team_name: "blackhats", team_lead: 4 },
    { id: 4, team_name: "Spies", team_lead: 1 }
  ],
  participant_team_members: [
    {
      team_member: 3,
      team_id: 1
    },
    {
      team_member: 4,
      team_id: 1
    },
    {
      team_member: 5,
      team_id: 1
    }
  ]
};
