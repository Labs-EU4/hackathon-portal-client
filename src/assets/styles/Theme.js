import * as color from './variables/colors';

const size = {
    mobileS: '320px',
    mobileM: '375px',
    mobileL: '425px',
    tablet: '768px',
    laptop: '1024px',
    laptopL: '1440px',
    desktop: '2560px'
};

export const theme = {
    color: {
        primary: {
            // light: color.primaryLight,
            regular: color.primary,
            // dark: color.primaryDark
        },
        secondary: {
            // light: color.secondaryLight,
            regular: color.secondary,
            // dark: color.secondaryDark
        },
        white: {
            regular: color.white,
        },
        black: {
            regular: color.black,
        },
        red: {
            regular: color.red,
            bright: color.brightRed
        },
        // grey: color.grey,
        grey: {
            regular: color.grey
        },
        green: {
            light: color.lightGreen,
            regular: color.green,
            dark: color.darkGreen
        },
    },
    shadow: {
        text: `text-shadow: .15px .15px 1px ${color.black}, .75px .5px 1.5px ${color.black}, .15px .15px 1px ${color.black};`,
        text_links: `1px .75px 1.5px ${color.black}`,
        box: `box-shadow: 0 3px 20px 0 rgba(0, 0, 0, .2)`,
        card: `box-shadow: 1px .5px 3px rgba(0, 0, 0, .15)`,
        intenseBox: `box-shadow: 0 2.5px 10px ${color.black};`,
        filter: `filter: drop-shadow(.5px .25px 1.5px ${color.black});
        filter: drop-shadow(.15px .15px 1px ${color.black}, .5px .25px 1.5px ${color.black}, .65px .4px 1px ${color.black});`,
        icon: `filter: drop-shadow(.5px .25px 1.5px ${color.black});
        filter: drop-shadow(25px .25px 10px ${color.black}, .5px .25px 1.5px ${color.black}, .65px .4px 1px ${color.black});`,
        btn: `2px 2px 3px ${color.black}`
    },
    flex: {
        center: 'display: flex; justify-content: center; align-items: center;',
        columnCenter: 'display: flex; flex-direction: column; justify-content: center; align-items: center;',
        column: 'display: flex; flex-direction: column;',

        custom: (justify='start', align='start', direction='row') => {
            return `
                display: flex; 
                flex-direction: ${direction}
                justify-content: ${justify}; 
                align-items: ${align};
            `
        },
    },
    device: {
        mobileS: `(min-width: ${size.mobileS})`,
        mobileM: `(min-width: ${size.mobileM})`,
        mobileL: `(min-width: ${size.mobileL})`,
        tablet: `(min-width: ${size.tablet})`,
        laptop: `(min-width: ${size.laptop})`,
        laptopL: `(min-width: ${size.laptopL})`,
        desktop: `(min-width: ${size.desktop})`,
        desktopL: `(min-width: ${size.desktop})`
    }
};