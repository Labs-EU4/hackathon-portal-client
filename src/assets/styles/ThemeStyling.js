import * as color from './variables/colors';

const size = {
    mobileS: '320px',
    mobileM: '375px',
    mobileL: '425px',
    tablet: '768px',
    laptop: '1024px',
    laptopL: '1440px',
    desktop: '2560px'
};

export const theme = {
    color: {
        primary: {
            light: '#81DBFD',
            regular: color.primary,
            // dark: color.primaryDark
        },
        secondary: {
            // light: color.secondaryLight,
            regular: color.secondary,
            // dark: color.secondaryDark
        },
        white: {
            regular: color.white,
            bg: color.bgWhite
        },
        black: {
            light: '#212121',
            regular: color.black,
        },
        red: {
            regular: color.red,
            bright: color.brightRed
        },
        blue: {
            light: color.lightBlue,
            regular: color.blue,
            bg: '#2196f3'
        },
        grey: {
            light: '#696969',
            regular: color.grey,
            dark: '#444',
            bg: color.bgGrey,
            border: 'rgba(14,19,24,.15)'
        },
        green: {
            light: color.lightGreen,
            regular: color.green,
            dark: color.darkGreen
        },
        success: color.success,
        warning: color.warning,
        danger: color.danger,
        info: color.info,
        link: {
            hover: 'rgba(14,19,24, .07)',
        }
    },
    fontSize: {
        smaller: "font-size: 1.2rem",
        small: "font-size: 1.4rem;",
        base: "font-size: 1.6rem;",
        medium: "font-size: 1.8rem",
        large: "font-size: 2rem",
        h1: "font-size: 7.5rem;",
        h2: "font-size: 5.5rem;",
        h3: "font-size: 3.5rem;",
        h4: "font-size: 2.5rem;",
        button: "font-size: 1.8rem;"
    },
    shadow: {
        text: `text-shadow: .5px .5px 1px ${color.black}`,
        text_links: `1px .75px 1.5px ${color.black}`,
        box: `box-shadow: 0 3px 20px 0 rgba(0, 0, 0, .2)`,
        card: `box-shadow: 1px .5px 3px rgba(0, 0, 0, .15)`,
        intenseBox: `box-shadow: 0 2.5px 10px ${color.black};`,
        filter: `filter: drop-shadow(.15px .15px 1.5px ${color.black});
        filter: drop-shadow(-.15px -.15px 1px ${color.black});`,
        btn: `2px 2px 3px ${color.black}`
    },
    flex: {
        center: 'display: flex; justify-content: center; align-items: center;',
        columnCenter: 'display: flex; flex-direction: column; justify-content: center; align-items: center;',
        column: 'display: flex; flex-direction: column;',

        custom: (justify='start', align='start', direction='row', wrap='no-wrap') => {
            return `
                display: flex; 
                flex-direction: ${direction}
                justify-content: ${justify}; 
                align-items: ${align};
                flex-wrap: ${wrap};
            `
        },
    },
    device: {
        mobileS: `(min-width: ${size.mobileS})`,
        mobileM: `(min-width: ${size.mobileM})`,
        mobileL: `(min-width: ${size.mobileL})`,
        tablet: `(min-width: ${size.tablet})`,
        laptop: `(min-width: ${size.laptop})`,
        laptopL: `(min-width: ${size.laptopL})`,
        desktop: `(min-width: ${size.desktop})`,
        desktopL: `(min-width: ${size.desktop})`
    }
};