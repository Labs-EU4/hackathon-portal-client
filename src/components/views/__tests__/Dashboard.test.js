import React from "react";
import { Router } from "react-router";
import { createMemoryHistory } from "history";
import { Provider } from "react-redux";
import { render, cleanup } from "@testing-library/react";
import configureStore from "redux-mock-store";
import Dashboard from "../Dashboard";
import { initialState } from "../../../utils/mockData";
import { theme } from "../../../assets/styles/ThemeStyling";
import { ThemeProvider } from "styled-components";

const history = createMemoryHistory();

afterEach(cleanup);

let mockStore;
let store;
let component;

beforeEach(() => {
  mockStore = configureStore();
  store = mockStore(initialState);

  component = render(
    <Router history={history}>
      <Provider store={store}>
        <ThemeProvider theme={theme}>
          <Dashboard />
        </ThemeProvider>
      </Provider>
    </Router>
  );
});

describe("It should render dashboard page correctly", () => {
  it("renders the Dashboard component correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
