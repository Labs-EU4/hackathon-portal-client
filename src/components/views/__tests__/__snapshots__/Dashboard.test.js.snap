// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render dashboard page correctly renders the Dashboard component correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="sc-bRBYWo bcWmiv"
        >
          <a
            class="sc-fMiknA dyzJBJ"
            href="/"
          >
            <img
              alt="Hackton - Organise hackathons"
              class="sc-jAaTju cZyUBe"
              src="Hackton-logo.png"
            />
          </a>
          <div
            class="sc-fBuWsC bPfjiF"
          >
            <div
              class="sc-hzDkRC itWopO"
            >
              <button
                class="sc-kEYyzF kbIspq sc-jhAzac itwdWK"
                color="grey"
                data-testid="label"
              >
                Log Out
              </button>
            </div>
          </div>
        </div>
        <div
          class="sc-iRbamj dyFMRm"
        >
          <div
            class="sc-jlyJG doWxFB sc-tilXH dZfDvR"
          >
            <div
              class="sc-bbmXgH crvuTG"
            >
              <h3
                class="sc-dnqmqq igidRh"
              >
                My hackathons
              </h3>
              <a
                href="/dashboard/new"
              >
                <button
                  class="sc-kEYyzF guLDtQ"
                  color="green"
                >
                  Create New
                </button>
              </a>
            </div>
            <div
              class="sc-jnlKLf jUEjEN"
              spacing="start"
            >
              <div
                class="sc-fAjcbJ cIAFYq"
              >
                <div
                  class="sc-dNLxif jbVVEK"
                >
                  <figure
                    class="sc-caSCKo dNANQe"
                  >
                    <img
                      alt="EuroHack"
                      src="event-img.jpg"
                    />
                  </figure>
                  <div
                    class="sc-cHGsZl YcHI"
                  >
                    <p
                      class="sc-cSHVUG sc-kgAjT jmNfKB"
                    >
                      Oct
                       
                      09
                       - 
                      Aug
                       
                      07
                      , 
                      2011
                    </p>
                    <h4
                      class="sc-jTzLTM iASFQm"
                    >
                      EuroHack
                    </h4>
                    <p
                      class="sc-cSHVUG sc-cJSrbW cTDtcZ"
                    >
                      Europe
                    </p>
                    <p
                      class="sc-cSHVUG fksUjT"
                    >
                      the best event hack hack hack macbook...
                    </p>
                    <div
                      class="sc-ksYbfQ ERwBz"
                    >
                      09/10/2011
                    </div>
                    <div
                      class="sc-TOsTZ jEmjRy"
                    >
                      <a
                        class="sc-kkGfuU jxuySM"
                        color="primary-reverse"
                        href="//event/1"
                      >
                        More Info
                      </a>
                      <a
                        class="sc-kkGfuU carJok"
                        color="primary"
                        href="/"
                      >
                        Join Event
                      </a>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-bookmark fa-w-12 sc-eqIVtm kKNrJl sc-hmzhuo iLRdnU"
                    data-icon="bookmark"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 384 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 512V48C0 21.49 21.49 0 48 0h288c26.51 0 48 21.49 48 48v464L192 400 0 512z"
                      fill="currentColor"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-star fa-w-18 sc-eqIVtm kKNrJl sc-frDJqD jwdodj"
                    data-icon="star"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 576 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="sc-fAjcbJ cIAFYq"
              >
                <div
                  class="sc-dNLxif jbVVEK"
                >
                  <figure
                    class="sc-caSCKo dNANQe"
                  >
                    <img
                      alt="World"
                      src="event-img.jpg"
                    />
                  </figure>
                  <div
                    class="sc-cHGsZl YcHI"
                  >
                    <p
                      class="sc-cSHVUG sc-kgAjT jmNfKB"
                    >
                      Jul
                       
                      09
                       - 
                      Aug
                       
                      07
                      , 
                      2011
                    </p>
                    <h4
                      class="sc-jTzLTM iASFQm"
                    >
                      World
                    </h4>
                    <p
                      class="sc-cSHVUG sc-cJSrbW cTDtcZ"
                    >
                      Asia
                    </p>
                    <p
                      class="sc-cSHVUG fksUjT"
                    >
                      number one hacker event in the world...
                    </p>
                    <div
                      class="sc-ksYbfQ ERwBz"
                    >
                      09/07/2011
                    </div>
                    <div
                      class="sc-TOsTZ jEmjRy"
                    >
                      <a
                        class="sc-kkGfuU jxuySM"
                        color="primary-reverse"
                        href="//event/2"
                      >
                        More Info
                      </a>
                      <a
                        class="sc-kkGfuU carJok"
                        color="primary"
                        href="/"
                      >
                        Join Event
                      </a>
                    </div>
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-bookmark fa-w-12 sc-eqIVtm kKNrJl sc-hmzhuo iLRdnU"
                    data-icon="bookmark"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 384 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 512V48C0 21.49 21.49 0 48 0h288c26.51 0 48 21.49 48 48v464L192 400 0 512z"
                      fill="currentColor"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-star fa-w-18 sc-eqIVtm kKNrJl sc-frDJqD jwdodj"
                    data-icon="star"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 576 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <div
              class="sc-bbmXgH crvuTG"
            >
              <h3
                class="sc-dnqmqq igidRh"
              >
                Global hackathons
              </h3>
            </div>
            <div
              class="sc-jnlKLf jUEjEN"
              spacing="start"
            />
          </div>
        </div>
        <div
          class="sc-bdVaJa jTDGAn"
        >
          <p
            class="sc-ifAKCX cvXEhe"
          >
            International Crafters Â© 2020
          </p>
          <div>
            <a
              class="sc-bwzfXH sc-bxivhb dvlUsV"
              href="#"
            >
              Privacy
            </a>
            <a
              class="sc-bwzfXH sc-bxivhb dvlUsV"
              href="#"
            >
              Contacts
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="sc-bRBYWo bcWmiv"
      >
        <a
          class="sc-fMiknA dyzJBJ"
          href="/"
        >
          <img
            alt="Hackton - Organise hackathons"
            class="sc-jAaTju cZyUBe"
            src="Hackton-logo.png"
          />
        </a>
        <div
          class="sc-fBuWsC bPfjiF"
        >
          <div
            class="sc-hzDkRC itWopO"
          >
            <button
              class="sc-kEYyzF kbIspq sc-jhAzac itwdWK"
              color="grey"
              data-testid="label"
            >
              Log Out
            </button>
          </div>
        </div>
      </div>
      <div
        class="sc-iRbamj dyFMRm"
      >
        <div
          class="sc-jlyJG doWxFB sc-tilXH dZfDvR"
        >
          <div
            class="sc-bbmXgH crvuTG"
          >
            <h3
              class="sc-dnqmqq igidRh"
            >
              My hackathons
            </h3>
            <a
              href="/dashboard/new"
            >
              <button
                class="sc-kEYyzF guLDtQ"
                color="green"
              >
                Create New
              </button>
            </a>
          </div>
          <div
            class="sc-jnlKLf jUEjEN"
            spacing="start"
          >
            <div
              class="sc-fAjcbJ cIAFYq"
            >
              <div
                class="sc-dNLxif jbVVEK"
              >
                <figure
                  class="sc-caSCKo dNANQe"
                >
                  <img
                    alt="EuroHack"
                    src="event-img.jpg"
                  />
                </figure>
                <div
                  class="sc-cHGsZl YcHI"
                >
                  <p
                    class="sc-cSHVUG sc-kgAjT jmNfKB"
                  >
                    Oct
                     
                    09
                     - 
                    Aug
                     
                    07
                    , 
                    2011
                  </p>
                  <h4
                    class="sc-jTzLTM iASFQm"
                  >
                    EuroHack
                  </h4>
                  <p
                    class="sc-cSHVUG sc-cJSrbW cTDtcZ"
                  >
                    Europe
                  </p>
                  <p
                    class="sc-cSHVUG fksUjT"
                  >
                    the best event hack hack hack macbook...
                  </p>
                  <div
                    class="sc-ksYbfQ ERwBz"
                  >
                    09/10/2011
                  </div>
                  <div
                    class="sc-TOsTZ jEmjRy"
                  >
                    <a
                      class="sc-kkGfuU jxuySM"
                      color="primary-reverse"
                      href="//event/1"
                    >
                      More Info
                    </a>
                    <a
                      class="sc-kkGfuU carJok"
                      color="primary"
                      href="/"
                    >
                      Join Event
                    </a>
                  </div>
                </div>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-bookmark fa-w-12 sc-eqIVtm kKNrJl sc-hmzhuo iLRdnU"
                  data-icon="bookmark"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 512V48C0 21.49 21.49 0 48 0h288c26.51 0 48 21.49 48 48v464L192 400 0 512z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star fa-w-18 sc-eqIVtm kKNrJl sc-frDJqD jwdodj"
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
            <div
              class="sc-fAjcbJ cIAFYq"
            >
              <div
                class="sc-dNLxif jbVVEK"
              >
                <figure
                  class="sc-caSCKo dNANQe"
                >
                  <img
                    alt="World"
                    src="event-img.jpg"
                  />
                </figure>
                <div
                  class="sc-cHGsZl YcHI"
                >
                  <p
                    class="sc-cSHVUG sc-kgAjT jmNfKB"
                  >
                    Jul
                     
                    09
                     - 
                    Aug
                     
                    07
                    , 
                    2011
                  </p>
                  <h4
                    class="sc-jTzLTM iASFQm"
                  >
                    World
                  </h4>
                  <p
                    class="sc-cSHVUG sc-cJSrbW cTDtcZ"
                  >
                    Asia
                  </p>
                  <p
                    class="sc-cSHVUG fksUjT"
                  >
                    number one hacker event in the world...
                  </p>
                  <div
                    class="sc-ksYbfQ ERwBz"
                  >
                    09/07/2011
                  </div>
                  <div
                    class="sc-TOsTZ jEmjRy"
                  >
                    <a
                      class="sc-kkGfuU jxuySM"
                      color="primary-reverse"
                      href="//event/2"
                    >
                      More Info
                    </a>
                    <a
                      class="sc-kkGfuU carJok"
                      color="primary"
                      href="/"
                    >
                      Join Event
                    </a>
                  </div>
                </div>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-bookmark fa-w-12 sc-eqIVtm kKNrJl sc-hmzhuo iLRdnU"
                  data-icon="bookmark"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 512V48C0 21.49 21.49 0 48 0h288c26.51 0 48 21.49 48 48v464L192 400 0 512z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star fa-w-18 sc-eqIVtm kKNrJl sc-frDJqD jwdodj"
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
          </div>
          <div
            class="sc-bbmXgH crvuTG"
          >
            <h3
              class="sc-dnqmqq igidRh"
            >
              Global hackathons
            </h3>
          </div>
          <div
            class="sc-jnlKLf jUEjEN"
            spacing="start"
          />
        </div>
      </div>
      <div
        class="sc-bdVaJa jTDGAn"
      >
        <p
          class="sc-ifAKCX cvXEhe"
        >
          International Crafters Â© 2020
        </p>
        <div>
          <a
            class="sc-bwzfXH sc-bxivhb dvlUsV"
            href="#"
          >
            Privacy
          </a>
          <a
            class="sc-bwzfXH sc-bxivhb dvlUsV"
            href="#"
          >
            Contacts
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
